imports/voting/ui/pages/SingleProposal.vue

```diff
@@ -123,58 +110,40 @@
         this.$store.commit('voting/setProposalType', upVote ? 'agree' : 'doubt');
         this.$store.dispatch('voting/voteByWeb3');
       },
-    }
-  };
+    },
+    meteor: {
+      $subscribe: {
+        'proposal'() {
+          return [
+            this.$route.params.id,
+          ];
+        },
+      },
+      proposal () {
+        return Proposals.findOne()
+      }
+    },
+  }
 </script>
 
 <style lang="scss">
   @import "/imports/core/ui/styles/variables";
-
-  .modal {
-    position: fixed;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    z-index: 100;
-    overflow-y: scroll;
-    padding: 40px 0;
-    .overlay {
-      position: fixed;
-      width: 100%;
-      height: 100%;
-      top: 0;
-      left: 0;
-      background: rgba(0, 0, 0, .7);
-      z-index: 1;
-    }
-    .modal-cont {
-      position: relative;
-      margin-bottom: 80px;
-      margin-left: auto;
-      margin-right: auto;
-      @media screen and (max-width: $container-width) {
-        max-width: 96vw;
-      }
-      z-index: 2;
-      width: 650px;
-      height: auto;
-      background: white;
-      border-radius: $base-border-radius;
-      box-shadow: $base-box-shadow;
-      .modal-header {
-        overflow: hidden;
-        button {
-          float: right;
-          border: 0;
-          padding: 0;
-        }
-      }
-      .modal-header, .voting-buttons, .voting-section, .current-status, .comments, .footer {
-        padding: $gutter $gutter * 2;
-      }
+  .single-proposal {
```

Comment:
Single proposal page should not be wider than proposal list page.

Reply:
@davidyuk OK I made changes, please help me to review again bro. I think my change for logo can fulfill what PR #25 do as well.